src/torrent_info.cpp

```diff
@@ -80,7 +80,7 @@ namespace libtorrent
 	} // anonymous namespace
 
 	// fixes invalid UTF-8 sequences
-	TORRENT_EXTRA_EXPORT bool verify_encoding(std::string& target)
+	bool verify_encoding(std::string& target)
```

Comment:
I believe this function has unit tests (if not, it should), which means the test need to be able to reach it